Tweak.xm

```diff
@@ -108,6 +90,25 @@ static CGFloat offset = 0;
 %end	// %hook SBUICAPackageView
 
 %hook SBFLockScreenDateView
+-(id)initWithFrame:(CGRect)arg1 {
```

Comment:
_Note:_ Keep the formatting in line with the rest of the code. Not necessary, but it looks nicer.
```- (id)initWithFrame:(CGRect)arg1 {``` (note the space between ```-``` and ```(id)``)